packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -407,7 +419,7 @@ const TrackListItemComponent = (props: TrackListItemComponentProps) => {
         <ListItemView
           style={styles.trackInnerContainer}
           onPress={isReorderable ? undefined : onPressTrack}
-          disabled={isDeleted || isLocked}
+          disabled={isDeleted || !isPlayable}
```

Comment:
does `!isPlayable` not imply `isDeleted`? should `isPlayable` factor in deleted state?

Reply:
not totally sure of desired ux, but before this PR we had a `!isDeleted && !isLocked` on line 419, and a `!isDeleted && isLocked` on line 441, so it seems like we use them separately.
however, tbh i'm not sure why line 441 is the way it is, seems to me that
a) we'd want the `halfTransparent` style for deleted tracks as well, and
b) we'd not want deleted tracks to appear in the playlist in the first place
Lmk your thoughts.

Reply:
Agree with A, but should check w/product

B is a product question. I'm pretty sure we have always shown deleted tracks in playlists, but I could see wanting to change that. Personally, I'd rather not see them when they're gone.

Reply:
checked against web, i believe deleted tracks should be half-transparent.